bio.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Pagina Mea</title>
+    </head>
+    <body>
+        <h1>Pagina mea</h1>
+        <ul type="square">
+            <li>nume:Dragos</li>
+            <li>prenume:Halaicu</li>
+        </ul>
+        <p>Salut,ma numesc Dragos sunt nascut in Botosani,locul in care am si copilarit.Clasele primare si gimnaziul le-am terminat in liceul Pedagogic Botosani dupa care in urma examinului de capacitate am ajuns in liceul Sportv din judet,am absolvit liceul in anul 2007 dupa care am pasit prin viata pana in zilele noastre.
+        </p>
+        <h2>Life kicks in</h2>
+        <p>Viata nu e chiar asa roz comparativ cu timpul varstei de 18 ani <strong><em> It is what it is</em></strong> nu vreau sa va plictisesc cu viata de anglia.Lucrez la o firma de taxi si ca si hobby am airsoftul pe care-l practic de 10 ani.</p>
+        <img src="C:\Users\halai\OneDrive\Desktop" alt="airsoft" width="200"height=300>
```

Comment:
poza ar trebuii adaugata in folderul cu tema si adaugat apoi ca src in imagine. 

Eu nu pot sa vad poza pentru ca nu am access la calculatorul tau sa o vad :D